src/unstar.js

```diff
@@ -35,23 +35,15 @@ Run \`ppm stars\` to see all your starred packages.\
           authorization: token
         }
       };
-      return new Promise((resolve, reject) => {
-        request.del(requestSettings, (error, response, body) => {
-          body ??= {};
-          if (error != null) {
-            this.logFailure();
-            return void reject(error);
-          }
-          if (response.statusCode !== 204) {
-            this.logFailure();
-            const message = request.getErrorMessage(body, error);
-            return void reject(`Unstarring package failed: ${message}`);
-          }
+      const response = await request.del(requestSettings).catch(error => { this.logFailure(); throw error; });
+      const body = response.body ?? {};
+      if (response.statusCode !== 204) {
+        this.logFailure();
+        const message = request.getErrorMessage(body, error);
```

Comment:
We will want to make sure to have the second parameter be `null` here, as `error` doesn't exist